wiki/Client/Beatmap_editor/Timing/en.md

```diff
@@ -1,25 +1,96 @@
+
+- Changing [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) on a certain section of the beatmap
+- Controlling the volume of [hitsounds](/wiki/Beatmapping/Hitsound)
+- Switching between active hitsounds' [samplesets](/wiki/Beatmapping/Sampleset)
+- Enabling or disabling [kiai time](/wiki/Gameplay/Kiai_time)
+
+Inherited timing points have green colour on the bottom timeline and in the [timing setup panel](#timing-setup-panel). Due to this, they are often called "green offsets", or "green lines".
+
+## Main view
+
+![Screenshot of the timing tab in the editor](/wiki/shared/timing/Timing_base.jpg)
+
+The timing tab can be opened by pressing `F3` and provides quick access to the following settings:
+
+| Name | Meaning |
+| :-- | :-- |
+| `BPM` | [Tempo](/wiki/Music_theory/Tempo) of the current timing point, measured in beats per minute (BPM). |
+| `Offset` | [Offset](/wiki/Offset#mapping) of the current timing point, measured in milliseconds. |
+| `Move already placed notes when changing the offset/BPM` | Keep hit objects attached to the [timeline](/wiki/Client/Beatmap_editor/Timelines) ticks during timing changes |
+| `Slider Velocity` | Base [slider velocity](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) of the whole beatmap |
+| `Slider Tick Rate` | How many [slider ticks](/wiki/Gameplay/Hit_object/Slider/Slider_tick) will be in a musical [beat](/wiki/Music_theory/Beat) |
+
+To quickly make larger or smaller changes, use the below modifiers:
+
+|  | Tempo | Offset | Slider velocity |
+| :-- | :--: | :--: | :--: |
+| `Ctrl` + click | 0.25 BPM | 1 ms | 1 [osu! pixel](/wiki/Client/Beatmap_editor/osu!_pixel) |
+| Default | 1 BPM | 2 ms | 10 osu! pixels |
+| `Shift` + click | 5 BPM | 10 ms | - |
+
+### Metronome
+
+![](img/metronome.png "The osu! metronome from the timing tab")
+
+The metronome, which is located in the upper right corner of the timing tab, helps quickly find approximate timing settings. The initial BPM and offset values of the song can be found by repeatedly tapping `T`, or clicking the `Tap Here!` button, while following the song's rhythm.
+
+The visual part of the metronome is tailored to the most popular [time signature](/wiki/Music_theory/Time_signature), 4/4. The first section denotes the [downbeat](/wiki/Music_theory/Downbeat) and flashes green, while remaining sections flash white to every beat of the song's [measures](/wiki/Music_theory/Measure). Uncommon signatures, such as 7/4, add extra flashes, but the size of the metronome doesn't change.
+
+## Timing setup panel
+
+![Screenshot of the timing setup panel](/wiki/shared/timing/TimingSetup.png)
+
+The timing setup panel, also called `Timing and Control Points`, is a pop-up window that can be opened by pressing `F6`. It contains several tabs for working with different options of the timing points, such as timing, [hitsounds](/wiki/Beatmapping/Hitsound) volume and [sampleset](/wiki/Beatmapping/Sampleset), or special effects.
```

Comment:
```suggestion
The timing setup panel, also called `Timing and Control Points`, is a pop-up window that can be opened by pressing `F6`. It contains several tabs for working with different options of the timing points, such as timing, [hitsounds](/wiki/Beatmapping/Hitsound), volumes, [samplesets](/wiki/Beatmapping/Sampleset), or special effects.
```

Reply:
your edit makes it sound like it's possible to add a standalone sample to a hit object from the timing panel directly, is that true?

Reply:
yes, if one specific green line's purpose is to effect one object then that is possible, its easy to see in keysounded mapsets